ui/installer/vCenterForWindows/install.bat

```diff
@@ -92,4 +79,12 @@ FOR /F "tokens=*" %%A IN (..\vCenterForWindows\_scratch_flags.txt) DO (
 cd ..\vCenterForWindows
 DEL _scratch_flags.txt
 
-ECHO VIC UI was successfully installed. Make sure to log out of vSphere Web Client if you are logged in, and log back in.
+IF /I %vic_ui_host_url% EQU NOURL (
+    ECHO =============================
+    ECHO With the current version of VIC, you have to manually copy the com.vmware.vicui.* folder in \ui\vsphere-client-serenity to %VMWARE_CFG_DIR%\vsphere-client\vc-packages\vsphere-client-serenity
+    ECHO Note: If you are running vCenter Server 5.5, copy the folder to %PROGRAMDATA%\VMware\vSphere Web Client\vc-packages\vsphere-client-serenity instead

```

Comment:
Never mind. I mistook the VIC version for VC version. Not sure how to get vCenter version. Will research if there's a way. It is still possible by running `vic-machine inspect` and extracting field `VMware OS version`, but that would require the user to put in the VCH's endpoint information which could be cumbersome